cmd/tests/cinder_csi_e2e_suite_test.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/config"
+	"github.com/onsi/ginkgo/reporters"
+	"github.com/onsi/gomega"
+	_ "k8s.io/cloud-provider-openstack/test"
```

Comment:
is this required?

Reply:
yes , its required to load, else gives error